frontend/src/services/dashboardConfigService.ts

```diff
@@ -7,6 +7,7 @@ export const fetchDashboardConfig = (): Promise<DashboardConfigKind> => {
     .get(url)
     .then((response) => response.data)
     .catch((e) => {
-      throw new Error(e.response.data.message);
+      // Throw the AxiosError with status code
+      throw e;
     });
```

Comment:
You can omit the catch and rethrow since this is the default behavior and there's no manipulation going on any longer with the error.